twilight-model/src/channel/message/reaction.rs

```diff
@@ -4,12 +4,18 @@ use serde::{Deserialize, Serialize};
 /// Reaction below a message.
 #[derive(Clone, Debug, Deserialize, Eq, Hash, PartialEq, Serialize)]
 pub struct Reaction {
+    /// HEX colors used for super reaction.
+    pub burst_colors: Vec<String>,
```

Comment:
First thought: surely we can store this more efficiently? Strings take up much more space than e.g. 3-element tuples of bytes

Reply:
A JDA dev asked (among other things) upstream about the color seralization too, without receiving a response.

Reply:
> First thought: surely we can store this more efficiently? Strings take up much more space than e.g. 3-element tuples of bytes

Implemented in [`472d52d` (#2275)](https://github.com/twilight-rs/twilight/pull/2275/commits/472d52dcc18bb6707e88fc3945f3c52dd9be945e)

> A JDA dev asked (among other things) upstream about the color seralization too, without receiving a response.

If I recall correctly, bots cannot add super reactions, so serialization shouldn't be a concern for the time being.